evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -306,3 +307,14 @@ def test_undersampler(X_y_binary):
     pipeline.predict(X)
     test = calculate_permutation_importance(pipeline, X, y, objective="Log Loss Binary")
     assert test is not None
+
+
+@pytest.mark.parametrize("estimator", [e for e in _all_estimators() if ('Classifier' in e.name and not any(s in e.name for s in ["Baseline", "Cat", "Elastic", "KN", "Ensemble"]))])
+def test_permutation_importance_oversampler(estimator, fraud_100):
+    pytest.importorskip('imblearn.over_sampling', reason='Skipping test because imbalanced-learn not installed')
+    X, y = fraud_100
+    pipeline = BinaryClassificationPipeline(component_graph=["Imputer", "One Hot Encoder", "DateTime Featurization Component", "SMOTENC Oversampler", estimator])
+    pipeline.fit(X=X, y=y)
+    pipeline.predict(X)
+    importance = calculate_permutation_importance(pipeline, X, y, objective="Log Loss Binary")
```

Comment:
@bchen1116 this can be a long-running computation. How long does this unit test take locally?

If its longer than a couple seconds total I suggest just doing it for one estimator. IMO I think that's acceptable because the goal for these tests is to prove that using a component which modifies the target during training (oversampler) doesn't break the model understanding tools, so iterating over the estimators is nice but not a requirement.

Reply:
Yep! Running all locally took ~36 seconds, while running with one estimator took ~4.5 seconds. Will change to using just 1 estimator!